squashfs-tools/squashfs-tools.json

```diff
@@ -1,8 +1,26 @@
+{
+    "name": "zstd",
+    "buildsystem": "meson",
+    "builddir": "true"
+    "subdir": "build/meson",
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://github.com/facebook/zstd/releases/download/v1.5.6/zstd-1.5.6.tar.gz",
```

Comment:
this is in Freedesktop SDK 24.08. And 1.5.5 is in 23.08

So you don't need it.

Reply:
Drop zstd build, thanks